### PR TITLE
Feature/VTN-26051 Add cover splash page + quickstart sections, integrate build tools

### PR DESCRIPTION
**Pull request by mikomonday**
Changes
- Webpack for building, module support
- Separated out js from `index.html`
- Added quickstart sections (4 in total)
- Added `_coverpage.md` with banner, quickstart links and whats new section (static)
- Added sass, separated out stylesheets

for dev: run `build:watch` AND `serve`

### CODE REVIEW

**Comment by mikomonday** [comment]
Merged in from master again today to resolve conflicts

File: package.json [hunk 1]
```diff
@@ -28,7 +32,19 @@
     "mocha-junit-reporter": "^1.18.0",
     "mocha-multi": "^1.0.1",
     "puppeteer": "^1.13.0",
+    "svg-inline-loader": "^0.8.0",
+    "webpack-cli": "^3.3.6",
     "shelljs": "^0.8.3",
     "yarn": "^1.15.2"
+  },
+  "dependencies": {
```
Comment (SteveShaffer) [inline package.json]: All these should be devDependencies

File: src/docs/_coverpage.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          <a href="/#/developer/applications/app-tutorial/" class="button-text">LEARN MORE ></a>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="footer">
+    <div class="wrapper">
+      <nav>
+        <ul>
+          <li><a href="#">Resources</a></li>
```
Comment (SteveShaffer) [inline src/docs/_coverpage.md]: Why do we have dead links?

File: src/docs/_coverpage.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+<!-- markdownlint-disable no-inline-html -->
+<div class="overview">
```
Comment (SteveShaffer) [inline src/docs/_coverpage.md]: We need to have some top-level link to like "home" or something.  We can't have the only front doors be through persona quickstarts and the links in "A Platform Approach"

**Comment by SteveShaffer** [comment]
We should strive to re-enable tests before we merge this.  There's a few legitimate failing tests in there right now related to this refactor.

File: src/docs/_coverpage.md (hunk 2 above)
Comment (mikomonday) [reply src/docs/_coverpage.md]: Fixed header with links from veritone.com as per meeting this morning

File: package.json (hunk 1 above)
Comment (mikomonday) [reply package.json]: Hm, did you move all of these already? I don't see a dependencies key anymore

File: src/docs/_coverpage.md (hunk 3 above)
Comment (mikomonday) [reply src/docs/_coverpage.md]: Added some links per Gus's recommendation.

**Comment by SteveShaffer** [comment]
The images on this page and sub-pages are broken: https://docs-i3md0atd9.now.sh/#/developer/applications/app-tutorial/

And the css tweaks are bleeding out into the sidebar.  But that's happening on master already.

Also, search is still disabled it seems?  Can we re-enable it?

And can we get a link to our npm site in there next to the GitHub link in the upper-right?
https://www.npmjs.com/org/veritone

And can we make the big Veritone logo in the splash page footer link to `veritone.com/`?  Cuz we don't have any links to veritone.com/ anywhere on that page atm.

And the support link is broken.  Should go to help.veritone.com, not www.help.veritone.com.  And arguably, it should just go to Slack.  Can we just remove it from this page?

And the apis and developer links along the top header are broken.

File: src/docs/_coverpage.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  <div class="action-bar">
+    <div class="wrapper">
+      <nav>
+        <ul>
+          <li><a href="http://local.veritone.com:3000/#/overview/"><h3>Overview</h3></a></li>
```
Comment (SteveShaffer) [inline src/docs/_coverpage.md]: Remove the local URL

Comment (mikomonday) [reply src/docs/_coverpage.md]: removed

**Comment by SteveShaffer** [comment]
Search is still broken when entering through the cover page.  But it's worth shipping without it.  I created VTN-28749 to track it.

**Pull request merged**